overview.md

```diff
@@ -51,19 +51,17 @@ A Catalog is a very simple construct - it just provides links to Items or to oth
 The closest analog is a folder in a file structure, it is the container for Items, but it can 
 also hold other containers (folders / catalogs). 
 
-The Collection specification extends the Catalog spec, so every Collection can serve as a catalog, 
-requiring the same fields. But it has a number of additional fields: license, extent (spatial and 
-temporal), providers, keywords and summaries. Every Item in a Collection links back to their Collection, 
-so clients can easily find fields like the license. Thus every Item implicitly shares the fields 
-described in their parent Collection.
+The Collection specification shares some fields with the catalog spec but has a number of additional fields:
+license, extent (spatial and temporal), providers, keywords and summaries. Every Item in a Collection links
+back to their Collection, so clients can easily find fields like the license.
```

Comment:
I'm ok with removing the bit about 'item implicitly shares the fields described in their parent collection', but I don't think it's really part of adding type field?